src/templates/render.ts

```diff
@@ -2,27 +2,29 @@ import { Controller } from "tsoa";
 import { Liquid } from "liquidjs";
 import { UniversalLoginSession } from "../adapters/interfaces/UniversalLoginSession";
 import { Env } from "../types";
+import {
+  layout,
```

Comment:
Moved these templates over to typescript. These are really internal so they don't need to be liquid templates. Might as well just be normal string templates or maybe lit templates? Or.. liquid templates if we do have it imported anyway.

Reply:
Why though? LiquidJs seemed a nice DX.  HTML in template strings isn't